cmd/puma-dev/main.go

```diff
@@ -7,26 +7,44 @@ import (
 	"runtime"
 )
 
-var fVersion = flag.Bool("V", false, "display version info")
+var (
+	EarlyExitClean = CommandResult{0, true}
```

Comment:
hmm, these should be lower case and declared as `const`

Reply:
hmm, actually go doesn't support const structs, so i think i'll leave it as is, unless there are objections.